ui_test/src/tests.rs

```diff
@@ -317,7 +324,13 @@ fn main() {
     let mut errors = vec![];
     check_annotations(messages, vec![], Path::new("moobar"), &mut errors, &config, "", &comments);
     match &errors[..] {
-        [Error::ErrorPatternWithoutErrorAnnotation(_, 5)] => {}
+        [Error::ErrorPatternWithoutErrorAnnotation(_, 5)] => Ok(()),
         _ => panic!("{:#?}", errors),
     }
 }
+
+static INIT: std::sync::Once = std::sync::Once::new();
+
+pub fn init() {
+    INIT.call_once(|| color_eyre::install().unwrap());
+}
```

Comment:
Uh, what is this? The static is de-facto even private to this function, right?

Reply:
yea, it's to make sure all tests have an initialized eyre, but only the first one (racy) actually performs the act